### PR TITLE
A  shocking PR

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
Adds a visual overlay for charge and organizes similar code

## About The Pull Request

- Adds a charge overlay, similar to the damage and healing overlays that appears when a weapon gains charge.
- Abnormalities and mobs that use charge (currently only shock centipede and gold dusk) also use this overlay.
- Fixes cane office weapons having the wrong path in some circumstances.
- Cleans up damage and healing overlays, as well as the stun overlay and puts them all in their own file.

## Why It's Good For The Game
Improv-ments galore!

## Changelog
:cl:
fix: fixed some cane office paths
imageadd: added charge icon for gaining charge
spellcheck: fixed a few typos
code: re-organized some code regarding damage and healing fx
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
I do like this PR.

File: code/game/objects/effects/temporary_visuals/effect_indicators.dm [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	icon_state = "dam_white"
+
+/obj/effect/temp_visual/damage_effect/black
+	icon_state = "dam_black"
+
+/obj/effect/temp_visual/damage_effect/pale
+	icon_state = "dam_pale"
+
+/obj/effect/temp_visual/damage_effect/pale
+	icon_state = "dam_pale"
```
Comment (Gboster-0) [inline code/game/objects/effects/temporary_visuals/effect_indicators.dm:44]: Failed copy-paste
```suggestion
/obj/effect/temp_visual/damage_effect/pale
	icon_state = "dam_pale"
```

Comment (Coxswain-Navigator) [reply code/game/objects/effects/temporary_visuals/effect_indicators.dm:44]: Good catch.

**Review by InsightfulParasite — APPROVED** [review]
Looks good to me unless another maint has a issue.

**Pull request merged**